i18n/ja.i18n.json

```diff
@@ -2,18 +2,18 @@
     "accept": "受け入れ",
     "act-activity-notify": "[Wekan] アクティビティ通知",
     "act-addAttachment": "__card__ に __attachment__ を添付しました",
-    "act-addChecklist": "added checklist __checklist__ to __card__",
-    "act-addChecklistItem": "added __checklistItem__ to checklist __checklist__ on __card__",
+    "act-addChecklist": "__card__ に __checklist__ を追加しました",
+    "act-addChecklistItem": "__checklist__ on __card__ に __checklistItem__ を追加しました",
     "act-addComment": "__card__: __comment__ をコメントしました",
     "act-createBoard": "__board__ を作成しました",
     "act-createCard": "__list__ に __card__ を追加しました",
-    "act-createCustomField": "created custom field __customField__",
+    "act-createCustomField": "__customField__ を作成しました",
```

Comment:
Only i18n/en.i18n.json has additions at GitHub, others are translated at https://transifex.com/wekan/wekan . Please remove other translation files. Here's how to remove commits:
https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-pull-request